packages/compiler/core/types.ts

```diff
@@ -315,6 +315,86 @@ export type Node =
   | Statement
   | Expression;
 
+export interface BindedCadlScriptNode extends CadlScriptNode, ContainerNode {
```

Comment:
```suggestion
export interface BoundCadlScriptNode extends CadlScriptNode, ContainerNode {
```

And s/Binded/Bound/ everywhere else.